### PR TITLE
docs: including CONTRIBUTING.md to the repo

### LINKED ISSUE #74: Include CONTRIBUTING.md to the repository
CONTRIBUTING.md will be included to the Repository to keep up with the current policies for source code management (SCM) system usage in Equinor.

### PR DESCRIPTION
**Pull request by kov117**

### CODE REVIEW

**Review by sefornes — APPROVED** [review]

**Comment by hknutsen** [comment]
@kov117 This repo does not contain any Terraform code, so having to run Terraform commands don't make sense to me 🤷‍♂️

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## CONTRIBUTING
+
+This repository is maintained by the [Ops team](https://github.com/orgs/equinor/teams/ops).
```
Comment (hknutsen) [inline CONTRIBUTING.md:3]: Equinor teams should not be referenced in public repositories.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+We use strict branch protection rules to ensure a [GitOps](https://www.redhat.com/en/topics/devops/what-is-gitops) approach to infrastructure provisioning.
```
Comment (hknutsen) [inline CONTRIBUTING.md:5]: We don't provision infrastructure from this repo.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Create a new branch.
+1. Run `terraform init`.
+1. Run `terraform validate`.
+1. Run `terraform plan`.
+1. Commit your changes.
+1. Create a pull request to trigger any relevant deployment workflows.
+1. Approve deployments.
+1. Request a review.
+1. Once approved, merge to branch `main`.
```
Comment (hknutsen) [inline CONTRIBUTING.md:17]: We don't run Terraform in this repo, so this doesn't make sense.

**Review by hknutsen — COMMENTED** [review]
See comments.

**Pull request merged**